src/select-menu/src/SelectMenuContent.js

```diff
@@ -81,14 +88,19 @@ export default class SelectMenuContent extends PureComponent {
               />
             </Pane>
           )}
-          <OptionsList
-            height={optionsListHeight}
-            hasFilter={hasFilter}
-            options={options}
-            isMultiSelect={isMultiSelect}
-            close={close}
-            {...listProps}
-          />
+
+          {options.length === 0 && hasEmptyView ? (
+            <Pane height={optionsListHeight}>{emptyView}</Pane>
```

Comment:
Should we push this deeper (into `OptionsList` itself)?

Reply:
No, because then `emptyView` would appear when there are no search results after filtering. There should be a separate prop for it.

Reply:
`emptyView` is useful for CTA views, for example when there menu is responsible for selecting warehouses and customer didn't connect any yet. Empty view could say that and show a button to go to connect a new warehouse. It would be incorrect to show the same view for empty search results.

Reply:
Yep, I agree with all of that. I was wondering if OptionsList should be responsible for those distinctions though. Sounds like that's not the case in your opinion.

Thanks!